main.js

```diff
@@ -68,6 +70,26 @@ module.exports = function(options) {
 	var sensuChecks = sensu(name, options.sensuChecks);
 	var healthChecks = options.healthChecks;
 
+	// Note: Only *report* CSP violations.
+	// The aim is to know which scripts are being executed via third-parties.
+	if (options.withContentSecurityPolicy) {
+		csp.extend(app, {
+			reportPolicy: {
+				directives: {
+					'report-uri': ['https://ft-next-csp-ingest.herokuapp.com/__csp-ingest'],
```

Comment:
No worries @matthew-andrews ... we're going to run this in reporting mode for a week or two just as an experiment, and if we decide to go ahead I'll sort out the csp-ingest URL.